src/ipc-wrapper/server.js

```diff
@@ -0,0 +1,39 @@
+// @ts-check
+import { createServer } from 'rpc-reflector'
+import { SubChannel } from './sub-channel.js'
+
+/**
+ * @param {import('../mapeo-manager.js').MapeoManager} manager
+ * @param {import('rpc-reflector/server.js').MessagePortLike} messagePort
```

Comment:
should this be accepting "plain" MessagePort instances too, similar to what rpc-reflector does? to account for usage in environments like the browser or electron, which adhere to what's described here: https://developer.mozilla.org/en-US/docs/Web/API/MessagePort

Node's worker_threads.MessagePort implementation does not match (interface + behavior), which is why things are slightly annoying.

improving interop would require some changes to event listeners similar to what's done here:

https://github.com/digidem/rpc-reflector/blob/3cbcc1d379b1bc7311cddded0d55fbc11319e3c4/client.js#L90-L92

also updating the SubChannel implementation